icinga2/resource_icinga2_host.go

```diff
@@ -51,6 +51,14 @@ func resourceIcinga2Host() *schema.Resource {
 					Type: schema.TypeString,
 				},
 			},
+			"groups": &schema.Schema{
```

Comment:
This is a duplicate definition of the groups attribute, starting at line 33-40. One of them will need to be removed.